sdk/src/Services/DynamoDBv2/Custom/DataModel/_async/Context.Async.cs

```diff
@@ -33,220 +33,164 @@ public partial class DynamoDBContext : IDynamoDBContext
+        }
+
+        /// <inheritdoc/>
+        public Task<T> LoadAsync<T>(T keyObject, CancellationToken cancellationToken = default)
+        {
+            return LoadHelperAsync(keyObject, new DynamoDBFlatConfig(null, Config), cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        [Obsolete("Use the LoadAsync overload that takes LoadConfig instead, since DynamoDBOperationConfig contains properties that are not applicable to LoadAsync.")]
+        public Task<T> LoadAsync<T>(T keyObject, DynamoDBOperationConfig operationConfig, CancellationToken cancellationToken = default)
+        {
+            return LoadHelperAsync(keyObject, new DynamoDBFlatConfig(operationConfig, Config), cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        public Task<T> LoadAsync<T>(T keyObject, LoadConfig loadConfig, CancellationToken cancellationToken = default)
         {
-            return LoadHelperAsync(keyObject, operationConfig, cancellationToken);
+            return LoadHelperAsync(keyObject, new DynamoDBFlatConfig(loadConfig?.ToDynamoDBOperationConfig(), Config), cancellationToken);
         }
 
         #endregion
 
         #region Delete async
 
-        /// <summary>
-        /// Initiates the asynchronous execution of the Delete operation.
-        /// </summary>
-        /// <typeparam name="T">Type of object.</typeparam>
-        /// <param name="value">Object to delete.</param>
-        /// <param name="cancellationToken">Token which can be used to cancel the task.</param>
-        /// <returns>A Task that can be used to poll or wait for results, or both.</returns>
-        public Task DeleteAsync<T>(T value, CancellationToken cancellationToken = default(CancellationToken))
+        /// <inheritdoc/>
+        public Task DeleteAsync<T>(T value, CancellationToken cancellationToken = default)
         {
-            return DeleteHelperAsync(value, null, cancellationToken);
+            return DeleteHelperAsync(value, new DynamoDBFlatConfig(null, Config), cancellationToken);
         }
 
-        /// <summary>
-        /// Initiates the asynchronous execution of the Delete operation.
-        /// </summary>
-        /// <typeparam name="T">Type of object.</typeparam>
-        /// <param name="value">Object to delete.</param>
-        /// <param name="operationConfig">Overriding configuration.</param>
-        /// <param name="cancellationToken">Token which can be used to cancel the task.</param>
-        /// <returns>A Task that can be used to poll or wait for results, or both.</returns>
-        public Task DeleteAsync<T>(T value, DynamoDBOperationConfig operationConfig, CancellationToken cancellationToken = default(CancellationToken))
+        /// <inheritdoc/>
+        [Obsolete("Use the DeleteAsync overload that takes LoadConfig instead, since DynamoDBOperationConfig contains properties that are not applicable to DeleteAsync.")]
```

Comment:
Wrong deprecation message - takes ~~LoadConfig~~ -> takes DeleteConfig

Reply:
Oops, fixed throughout in https://github.com/aws/aws-sdk-net/pull/3387/commits/1e8128085c97ee7a5f6d0803be53f48078bd4e05